src/browser/Navigators/Navigator/WebView/MozBrowserFrame.js

```diff
@@ -185,7 +185,7 @@ const decodeLocationChange =
     , canGoForward: null
     }
   : { type: "LocationChanged"
-    , uri: detail.uri
+    , url: detail.url
```

Comment:
@paulrouget Ok! That makes sense :) I noticed other lines in this file like this `uri: detail`, should I change those too?